presto-docs/src/main/sphinx/sql/describe-input.rst

```diff
@@ -0,0 +1,49 @@
+.. _describe-input:
+
+==============
+DESCRIBE INPUT
+==============
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    DESCRIBE INPUT statement_name
+
+Description
+-----------
+
+Describes the input parameters for a prepared statement.  It returns a table
+with the position and type of each parameter.  If the type of the parameter can
+not be determined, unknown is returned. For more information on preparing a
+statement see :ref:`prepare`.
+
+Examples
+--------
+
+1. Describe query with 3 parameters:
```

Comment:
Using a numbered list here is a bit strange. Let's write this section like this:
```
Examples
--------

Prepare and describe a query with three parameters:

.. code-block:: sql

    PREPARE my_select FROM
    SELECT ? FROM nation WHERE regionkey = ? AND name < ?

.. code-block:: sql
    
    DESCRIBE my_select

.. code-block:: none

     Position | Type
    --------------------
            0 | unknown
            1 | bigint
            2 | varchar
    (3 rows)

Describe a query with no parameters: